ui/v2.5/src/components/Settings/Tasks/DataManagementTasks.tsx

```diff
@@ -507,6 +556,69 @@ export const DataManagementTasks: React.FC<IDataManagementTasks> = ({
             <FormattedMessage id="actions.rename_gen_files" />
           </Button>
         </Setting>
+
+        <div className="setting-group">
+          <Setting
+            headingID="actions.migrate_blobs"
+            subHeadingID="config.tasks.migrate_blobs.description"
+          >
+            <Button
+              id="migrateBlobs"
+              variant="danger"
+              onClick={() => onMigrateBlobs()}
+            >
+              <FormattedMessage id="actions.migrate_blobs" />
+            </Button>
+          </Setting>
+
+          <BooleanSetting
+            id="migrate-blobs-delete-old"
+            checked={migrateBlobsOptions.deleteOld ?? false}
+            headingID="config.tasks.migrate_blobs.delete_old"
+            onChange={(v) =>
+              setMigrateBlobsOptions({ ...migrateBlobsOptions, deleteOld: v })
+            }
+          />
+        </div>
+
+        <div className="setting-group">
+          <Setting
+            headingID="actions.migrate_scene_screenshots"
+            subHeadingID="config.tasks.migrate_scene_screenshots.description"
+          >
+            <Button
+              id="migrateSceneScreenshots"
+              variant="danger"
+              onClick={() => onMigrateSceneScreenshots()}
+            >
+              <FormattedMessage id="actions.migrate_scene_screenshots" />
+            </Button>
+          </Setting>
+
+          <BooleanSetting
+            id="migrate-blobs-delete-files"
```

Comment:
is the id here correct? it is the same as the next Booleansetting and it might explain the delete screenshot files toggle not working correctly for me
![Screenshot_2023-03-06-23-13-49-857-edit_com android chrome](https://user-images.githubusercontent.com/48220860/223277140-07b30246-37b9-482f-966f-659ab96271e3.jpg)
( actual screenshot was before latest 2 commits but behaviour is the same after last commit)